docusaurus/docs-internals/editor-controls/multivalue-controls/corners.md

```diff
@@ -0,0 +1,177 @@
+| :-------------- | :--------------------------------------- | :-----: | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `id`            | `string`                                 |    -    | The identifier of the key where the corners will save your data                                                                                                                                                                  |
+| `type`          | `string`                                 |    -    | Type should be `"corners"` to use this control                                                                                                                                                                                   |
+| `label?`        | `string`                                 |    -    | The label displayed on the left side of the control                                                                                                                                                                              |
+| `position?`     | `number`                                 |    -    | The position of the control in toolbar                                                                                                                                                                                           |
+| `devices?`      | `"all"` \| `"desktop"` \| `"responsive"` | `"all"` | Define the devices where the control will be rendered. `"all"` renders the control on all devices. `"desktop"` renders the control only on desktop devices. `"responsive"` renders the control on both tablet and mobile devices |
+| `disabled?`     | `boolean`                                | `false` | Configure the condition under which the control is disabled or enabled                                                                                                                                                           |
+| `selector?`     | `string`                                 |    -    | Selector that determines the target element to which the corners styling will be applied                                                                                                                                         |
+| `config?.units` | `Array<Unit>`                            |  `[ ]`  | The units of measurement displayed next to the slider value. Accepts all possible CSS units (`"px"`, `"%"`) and your custom values <br/> <br/> <b>`Unit = "px" \| "%"`</b> <br/> <br/>value of units - a valid CSS unit          |
```

Comment:
change description of `units` because as i see from types, unit can takes only px and %:
![image](https://github.com/EasyBrizy/Brizy-Local-Editor/assets/52622998/bd67c047-97ed-404b-885d-d37c58ab69da)